enginetest/queries/alter_table_queries.go

```diff
@@ -423,6 +423,105 @@ var AlterTableScripts = []ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "ALTER TABLE does not change column collations",
```

Comment:
nit: script tests with a bunch of parallel things being tested are harder to debug than ex: 4 script tests, or interleaving create/alter/show queries